g11n-ws/vip-manager-l10n/src/main/java/com/vmware/l10n/conf/CollectSourceValidationInterceptor.java

```diff
@@ -237,7 +237,6 @@ private static void validateAllowList(HttpServletRequest request, Map<String, Li
 		}
 		if(!allowList.isEmpty() && allowList.containsKey(productName)
 				&& allowList.get(productName).contains(version)) {
-			return;
 		}else {
```

Comment:
why remove the return？

Reply:
It's unnecessary.

Reply:
Add return can save CPU resources

Reply:
There is no other code after that. right?